cypress/e2e/auth/auth.cy.js

```diff
@@ -0,0 +1,138 @@
+const { getRandomEmail, getRandomString } = require("../../../utilites/helper");
+const loginPagePO = require("../../page_objects/login_PO");
+
+describe("authentication module", function () {
+  beforeEach(function () {
+    loginPagePO.visitLoginPage();
+  });
+  it("URL should be https://frontendbootcamp.proshore.eu/accounts/login", function () {
+    cy.url().should(
+      "eq",
+      "https://frontendbootcamp.proshore.eu/accounts/login"
+    );
+  });
+  it("verify valid login", function () {
+    cy.login();
+  });
+
+  it(" should not be submit with empty details", function () {
+    loginPagePO.clickLogin();
+    loginPagePO.elements.loginBtn().should("be.disabled");
+  });
+
+  it("Verify the validation msg when login is attempted with less than 6 letter password", function () {
+    const randomGenerateEmail = getRandomEmail();
+    const randomGenerateName = getRandomString(3);
+    loginPagePO.typeEmail(randomGenerateEmail);
+    loginPagePO.typePassword(randomGenerateName);
+    loginPagePO.clickLogin();
+  loginPagePO.verifyPasswordLengthErrorMessage();
+
+   
+  });
+  it("Verify the login process with invalidLogin credentials ", function () {
+    const randomGenerateEmail = getRandomEmail();
+    const randomGenerateName = getRandomString(7);
+    // cy.log(randomGenerateName)
+    // cy.pause()
+    loginPagePO.typeEmail(randomGenerateEmail);
+    loginPagePO.typePassword(randomGenerateName);
+    loginPagePO.clickLogin();
+    loginPagePO.verifyInvalidCredentialAlertMessage();
+  });
+  it("should visible the password", function () {
+    cy.login();
+    cy.get("#showPassword").click();
+    cy.get("#password").invoke("attr", "type").should("eq", "text");
+  });
+  it("It should not be submit with empty email address", function () {
+    const randomGenerateName = getRandomString(6);
+    loginPagePO.typePassword(randomGenerateName);
+    loginPagePO.clickLogin();
+    loginPagePO.elements.loginBtn().should("be.disabled");
+  });
+  it(" should not be submit with empty password", function () {
+    const randomGenerateEmail = getRandomEmail();
+    loginPagePO.typeEmail(randomGenerateEmail);
+    loginPagePO.clickLogin();
+    loginPagePO.elements.loginBtn().should("be.disabled");
+  });
+  it("It should not be submit with incorrect password", function () {
+    const randomGenerateEmail = getRandomEmail();
+    const randomGenerateName = getRandomString();
```

Comment:
the description and assertions does not go in hand
please use password more than 6 letters